src/App.tsx

```diff
@@ -50,8 +50,10 @@ const App: React.FC = () => {
       setSwaggerDocs(swaggerDoc);
       const appInfo = await AppInfoService.getAppInfo();
       setAppInfo(appInfo);
-      const userInfo = await userService.findOne(appInfo?.userId);
-      setUserInfo(userInfo);
+      if (appInfo.userId) {
```

Comment:
can `appInfo` object be undefined or null here? if so, check for `userId` will fail